controllers/admin/AdminDPDBalticsAjaxShipmentsController.php

```diff
@@ -81,6 +81,7 @@ public function postProcess()
                 $response['status'] = false;
                 $shipmentId = (int)Tools::getValue('shipment_id');
                 $this->printReturnLabel($shipmentId);
+                die();
```

Comment:
printReturnLabel already has exit for all cases if i understand correctly, this die() is not needed.

Reply:
left it just in case. because before some 1 changed the function logic and left one case without exit or die and shit happen, because module is not that new i will leave double check to make sure it exits :D